src/NuGet.Core/NuGet.Packaging/PackageArchiveReader.cs

```diff
@@ -452,6 +452,10 @@ public override bool CanVerifySignedPackages(SignedPackageVerifierSettings verif
             {
                 return false;
             }
+            else if (RuntimeEnvironmentHelper.IsLinux || RuntimeEnvironmentHelper.IsMacOSX)
```

Comment:
I have a question, are we going to disable verification on Linux only, or on both Linux and MacOS?